source/features/repo-age.tsx

```diff
@@ -43,11 +43,13 @@ const getRepoAge = async (commitSha: string, commitsCount: number): Promise<[str
 };
 
 const getFirstCommit = cache.function(async (): Promise<[string, string] | undefined> => {
-	const commitInfo = await elementReady<HTMLAnchorElement | HTMLScriptElement>('a.commit-tease-sha, include-fragment.commit-tease');
+	const commitInfo = await elementReady<HTMLAnchorElement | HTMLScriptElement>('.commit-tease a[href*="/commit/"]');
```

Comment:
The selector uses `include-fragment` because if the current view isn't `master`, GitHub will load it later. Try for example going to https://github.com/sindresorhus/refined-github and loading any tag or specific commit from the branch selector. You can also verify via `view-source:` on those pages.